fivex/api/__init__.py

```diff
@@ -26,16 +26,19 @@ def region_query(chrom, start, end):
     """
     tissue = request.args.get("tissue", None)
     gene_id = request.args.get("gene_id", None)
+    study = request.args.get("study", None)
     piponly = request.args.get("piponly", None)
 
     data = [
         res.to_dict()
         for res in query_variants(
             chrom=chrom,
             start=start,
+            rowstoskip=1,  # Region query uses the original EBi data files, which all have a header row
```

Comment:
Will older datasets (eg GTEx) be using a different format?

For now this is ok. Long term, it would be nice if the ingest process standardized the differences across file formats. Then, the display/query code wouldn't have to handle differences in file options.

Reply:
The EBI data set is a superset of data containing GTEx. The file format is different from that used in our GTEx-only browser, making two data sets incompatible in general.

Another difference is that the EBI data set will be getting future updates, while the GTEx data set is essentially finalized, so I think it makes more sense to adapt to processing the new EBI-formatted data sets going forward.

The downside is that the supporting files which rely on PIPs (based on DAP-G) will need to change, since DAP-G was not used to analyze all the other data sets, but PIPs are a key part of the browser which makes it much easier to pick out potentially interesting variants, tissues, and genes.

Reply:
> The downside is that the supporting files which rely on PIPs (based on DAP-G) will need to change, since DAP-G was not used to analyze all the other data sets, but PIPs are a key part of the browser which makes it much easier to pick out potentially interesting variants, tissues, and genes.

Ooof, that's a good point.

How do we want to handle the UI in that case? ("key feature that sometimes can't be used")

Reply:
As currently written, when PIPs are not available, the default behavior is to mark them as missing - this is true for the majority of the points even in GTEx. Currently we should maintain that behavior in the plots until we can find a suitable replacement statistic to use. Thanks to the way we wrote PipAdder, we should be able to plug in a new database to add PIP (or any other metric) to the EBI data.

In a related issue, we will need to generate new data to support the data tables on the bottom of both the single variant page and region page.